lmdeploy/vl/utils.py

```diff
@@ -22,13 +22,16 @@ def load_image_from_base64(image: Union[bytes, str]) -> Image.Image:
 def load_image(image_url: str) -> Image.Image:
     """load image from url, local path or openai GPT4V."""
 
+    FETCH_TIMEOUT = 1
```

Comment:
I am afraid 1s is too short, maybe 10s. Can we use a environment value?

Reply:
I mean the `os.environ` way. @irexyc any ideas?

Reply:
no problem! change was pushed.

Reply:
Could you please add a FAQ section in the user guide, introducing this environment variable?
https://github.com/InternLM/lmdeploy/blob/main/docs/en/serving/api_server_vl.md
https://github.com/InternLM/lmdeploy/blob/main/docs/zh_cn/serving/api_server_vl.md

Reply:
FAQ is updated, please check that the `zh` docs make sense as I am using translate.

Reply:
The translation is great :)